### DISCUSSION TITLE
梅林380点击提交无反应

### DISCUSSION THREAD

**Discussion by sailing178** [messense/aliyundrive-webdav#170] [Q&A]
![image](https://user-images.githubusercontent.com/89731041/143679267-4586b513-25bc-49bd-a0af-30e1212b5dba.png)

**Comment by sailing178** [comment]
Nov 27 19:17:39 admin: [软件中心]: 完成 aliyundrivewebdav 安装
Nov 27 19:17:41 skipd[499]: sync tick
Nov 27 19:21:11 skipd[499]: sync tick
Nov 27 19:21:14 skipd[499]: sync tick
Nov 27 19:23:23 skipd[499]: sync tick
Nov 27 19:23:26 skipd[499]: sync tick
Nov 27 19:36:25 rc_service: httpd 517:notify_rc restart_wlcscan
Nov 27 19:36:47 rc_service: httpd 517:notify_rc restart_qos;restart_firewall
Nov 27 19:36:48 kernel: Ebtables v2.0 registered
Nov 27 19:36:51 start_nat_rules: apply the nat_rules(/tmp/nat_rules_eth0_eth0)!
Nov 27 19:36:51 custom script: Running /jffs/scripts/nat-start
Nov 27 19:36:52 start_nat_rules: apply the nat_rules(/tmp/nat_rules_eth0_eth0)!
Nov 27 19:36:52 custom script: Running /jffs/scripts/nat-start

**Comment by messense** [comment]
没填 refresh token？
